src/api/wine_api.tsx

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+import { getWinesUrl } from "./base_urls";
+
+export const fetchAllWines = async () => {
```

Comment:
I was under the impression we do not test functions that make API calls, just mock them. Since `<App />` doesn't take in a `prop` of fetchAllWines I didn't mock it, but instead used `msw` which intercepts any API calls `<App />` makes and returns a mocked response.

Reply:
Yeah, let's bookmark this for a discussion. I'd be curious to hear what @cdemyanovich and @k3ntako have to say. I usually take the approach to test configuration, so like you're using msw, which simplifies your tests and that's great, but how do you know that `fetchAllWines` is hitting the correct endpoint. you could've typed it wrong or not used the constant. having a unit test for that, along with various headers and other things you might have to configured the request gives you an additional layer of confidence that you're code is working as expected.